packages/react-components/react-progress/src/utils/clampMax.ts

```diff
@@ -0,0 +1,11 @@
+export const clampMax = (max?: number) => {
```

Comment:
You can guarantee that `max` is not `undefined` at this point:

```suggestion
export const clampMax = (max: number): number => {
```